articles/azure-arc/servers/includes/network-requirements.md

```diff
@@ -13,7 +13,7 @@ The Azure Connected Machine agent for Linux and Windows communicates outbound se
 To further secure your network connectivity to Azure Arc, instead of using public networks and proxy servers, you can implement an [Azure Arc Private Link Scope](../private-link-security.md) .
 
 > [!NOTE]
-> Azure Arc-enabled servers does not support using a [Log Analytics gateway](../../../azure-monitor/agents/gateway.md) as a proxy for the Connected Machine agent.
+> Azure Arc-enabled servers does not support using a [Log Analytics gateway](../../../azure-monitor/agents/gateway.md) as a proxy for the Connected Machine agent.If outbound connectivity is restricted by your firewall or proxy server, make sure the URLs and Service Tags listed below are not blocked.At the same time AMA agent supports Log Analytics gateway.
```

Comment:
@kausd1 @Court72 thanks for the suggestions. I prefer that the note not be combined with the sentence after it. Instead, please add the following to the note: At the same time, Azure Monitor Agent supports Log Analytics gateway.